system/lib/libcxxabi/src/cxa_exception_emscripten.cpp

```diff
@@ -136,12 +137,12 @@ void __cxa_decrement_exception_refcount(void *thrown_object) _NOEXCEPT {
     if (thrown_object != NULL )
     {
         __cxa_exception* exception_header = cxa_exception_from_thrown_object(thrown_object);
-        DEBUG("DEC: %p refcnt=%zu rethrown=%d\n", thrown_object,
+        DEBUG("DEC: %p refcnt=%zu rethrown=%d", thrown_object,
               exception_header->referenceCount, exception_header->rethrown);
         assert(exception_header->referenceCount > 0);
         if (std::__libcpp_atomic_add(&exception_header->referenceCount, size_t(-1)) == 0 && !exception_header->rethrown)
         {
-            DEBUG("DEL: %p\n", thrown_object);
+            DEBUG("DEL: %p (dtor=%p)", thrown_object, exception_header->exceptionDestructor);
             if (NULL != exception_header->exceptionDestructor)
```

Comment:
How did pointer comparison like this even compile without the correct signature :/